src/matter/cluster/WaterContentMeasurementCluster.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {Attribute, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse} from "./Cluster";
+import {BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8} from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for WaterContent Measurement.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.6
+ */
+/** Percentage of water in the air */
```

Comment:
Please have only one comment , so maybe move to the top and also repeat the "full comment" with the li nk reference for each of the cluster definitions